client.go

```diff
@@ -561,6 +561,12 @@ type EntityErrors struct {
 	warns []error
 }
 
+// NewEntityErrors returns a wrapper for errors encountered while parsing
+// entity struct tags.
+func NewEntityErrors(warns []error) *EntityErrors {
```

Comment:
Because it may contain multiple errors :) but, see the [`FindEntities`](https://github.com/uber-go/dosa/blob/master/finder.go#L41) method which generates these errors.